ios/FluentUI/Drawer/DrawerTokens.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
```

Comment:
why do we need this?

Reply:
I have removed this, it's no longer needed

Reply:
I changed this to UIKit as CI complaints about use fo `CGfloat` and `UIColor`. Seems like it doesn't inherit the framework from base token.